src/components/scheduling/goals/SchedulingGoalForm.svelte

```diff
@@ -260,7 +288,7 @@
         <select bind:value={goalModelId} class="st-select w-100" name="model">
```

Comment:
I think it might be good to also have permissions on the select elements themselves for when the role is something like "viewer"

Reply:
Ah yes! agree, I meant to circle back to the handle all the inputs, just slipped my mind from focusing on other details.

Reply:
Ok I just pushed up more detailed permissions controls for the dropdowns and inputs.